public/docs/ts/latest/tutorial/index.jade

```diff
@@ -2,80 +2,82 @@ block includes
+  You'll use one-way data binding for read-only data. You'll add editable fields to update a model
+  with two-way data binding. You'll bind component methods to user events, like keystrokes and clicks.
+  You'll enable users to select a hero from a master list and edit that hero in the details view. You'll
+  format data with pipes. You'll create a shared service to assemble the heroes.
+  And you'll use routing to navigate among different views and their components.
+  <!-- CF: Should this be a bullet list? -->
 
-  We’ll learn enough core Angular to get started and gain confidence that
-  Angular can do whatever we need it to do.
-  We'll be covering a lot of ground at an introductory level but we’ll find plenty of links
-  to chapters with greater depth.
+  You'll learn enough core Angular to get started and gain confidence that
+  Angular can do whatever you need it to do.
+  You'll cover a lot of ground at an introductory level, and you'll find many links
+  to pages with greater depth.
+
+  When you're done with this tutorial, the app will look like this <live-example name="toh-6"></live-example>.
 
-  Run the <live-example name="toh-6"></live-example>.
 
 .l-main-section
 :marked
-  ## The End Game
+  ## The end game
 
-  Here's a visual idea of where we're going in this tour, beginning with the "Dashboard"
-  view and our most heroic heroes:
+  Here's a visual idea of where this tutorial leads, beginning with the "Dashboard"
+  view and the most heroic heroes:
 
 figure.image-display
   img(src='/resources/images/devguide/toh/heroes-dashboard-1.png' alt="Output of heroes dashboard")
 
 :marked
-  Above the dashboard are two links ("Dashboard" and "Heroes").
-  We could click them to navigate between this Dashboard and a Heroes view.
+  Users can click the two links above the dashboard ("Dashboard" and "Heroes") 
+  to navigate between this Dashboard view and a Heroes view.
 
-  Instead we click the dashboard hero named "Magneta" and the router takes us to a "Hero Details" view
-  of that hero where we can change the hero's name.
+  If a user clicks the dashboard hero "Magneta," the router opens a "Hero Details" view
+  where the user can change the hero's name.
 
 figure.image-display
   img(src='/resources/images/devguide/toh/hero-details-1.png' alt="Details of hero in app")
 
 :marked
-  Clicking the "Back" button would return us to the "Dashboard".
-  Links at the top can take us to either of the main views.
-  We'll click "Heroes". The app takes to the "Heroes" master list view.
+  Clicking the "Back" button returns users to the Dashboard.
+  Links at the top take them to either of the main views.
+  If a user clicks "Heroes," the app displays the "Heroes" master list view.
 
 figure.image-display
   img(src='/resources/images/devguide/toh/heroes-list-2.png' alt="Output of heroes list app")
+//  CF: The ability to add heroes isn't shown in the images or discussed in this page. Should that be added?
```

Comment:
I made a note in the "Todos" at the top of this PR that we need to review the images and the current state of the ToH app. These images were taken before ToH 6.